app/controllers/subscriptions_controller.rb

```diff
@@ -28,6 +28,20 @@ def create
     render json: { id: subscription.id }, status: status
   end
 
+  def index
+    subscriber = find_subscriber
+
+    return render_not_found if subscriber.nil?
+
+    subscriptions = Subscription.active.
+      joins(:subscriber_list).
+      where(subscriber: subscriber).
+      order('subscriber_lists.title').
+      select(:id, :'subscriber_lists.title', :frequency).as_json
+
+    render json: { subscriber_address: address, subscriptions: subscriptions }
```

Comment:
Instead of showing the address here, it might be good to just have a `subscriber: subscriber.to_json` then we can add extra fields easily if we want to.

Reply:
I've changed it to return the full subscriber